### PR TITLE
Add community blog note

### PR DESCRIPTION
**Pull request by merveenoyan**
cc @pcuenca @osanseviero

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # The Hugging Face Blog Repository 🤗
 This is the official repository of the [Hugging Face Blog](https://hf.co/blog). 
 
+**If you are an external contributor**: If your blog is not a collaboration blog with Hugging Face, please consider creating a [community blog](https://huggingface.co/blog-explorers) instead.
```
Comment (osanseviero) [inline README.md:4]:
```suggestion
**If you are an external contributor**: If your blog post is not a collaboration post with Hugging Face, please consider creating a [community blog](https://huggingface.co/blog-explorers) instead.
```

**Review by osanseviero — COMMENTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (pcuenca) [inline README.md]:
```suggestion
**If you are an external contributor**: If your post is not a collaboration with Hugging Face, please consider creating a community blog instead by joining [this organization](https://huggingface.co/blog-explorers) with your Hugging Face user account.
```

Minor suggestion.

Comment (julien-c) [reply README.md]: maybe add a screenshot of where the community blogs are displayed (though it might change in the future, so idk)

**Review by pcuenca — APPROVED** [review]
I think Julien's screenshot idea would be fine too :)

**Pull request merged**